instrumentation/base/lib/opentelemetry/instrumentation/base.rb

```diff
@@ -303,6 +320,46 @@ def enabled_by_env_var?
         end
         ENV[var_name] != 'false'
       end
+
+      def config_overrides_from_env
+        var_name = name.dup.tap do |n|
+          n.upcase!
+          n.gsub!('::', '_')
+          n.gsub!('OPENTELEMETRY_', 'OTEL_RUBY_')
+          n << '_CONFIG_OPTS'
+        end
+
+        environment_config_overrides = {}
+        env_config_options = ENV[var_name]&.split(';')
+
+        return environment_config_overrides if env_config_options.nil?
+
+        env_config_options.each_with_object(environment_config_overrides) do |env_config_option, eco|
+          parts = env_config_option.split('=')
+          option_name = parts[0].to_sym
+          eco[option_name] = parts[1]
+        end
+
+        environment_config_overrides
+      end
+
+      def coerce_env_var(env_var, validation_type) # rubocop:disable Metrics/CyclomaticComplexity
+        case validation_type
+        when :array
+          env_var.split(',').map(&:strip)
+        when :boolean
+          env_var.to_s.strip.downcase == 'true'
```

Comment:
What happens is the user provides a random word here?  Will it evaluate to false?  

Could we add a test here for that?

Reply:
yup the intention is to (evaluate to false). will add test to confirm